Philips.CodeAnalysis.Test/Verifiers/AssertCodeFixVerifier.cs

```diff
@@ -60,19 +60,19 @@ protected void VerifyError(string methodBody, string methodAttributes, int expec
 				expected.Message = new Regex(error, RegexOptions.Singleline, TimeSpan.FromSeconds(1));
 			}
 
-			VerifyDiagnostic(test, expected);
+			await VerifyDiagnostic(test, expected).ConfigureAwait(false);
 		}
 
-		protected void VerifyError(string methodBody, int expectedErrorLineOffset = 0, int expectedErrorColumnOffset = 0, string error = null)
+		protected async Task VerifyErrorAsync(string methodBody, int expectedErrorLineOffset = 0, int expectedErrorColumnOffset = 0, string error = null)
```

Comment:
This method has the Async postfix, please remove it

Reply:
done